docs/resources/ip.md

```diff
@@ -33,10 +33,10 @@ resource "katapult_ip" "primary-db" {
 
 ### Optional
 
-- `label` (String) VIP label. Required when **vip** is `true`.
+- `label` (String) VIP label.Required when **vip** is `true`.
 - `network_id` (String)
-- `version` (Number) IPv4 or IPv6. Defaults to `4`.
-- `vip` (Boolean) Defaults to `false`.
+- `version` (Number) IPv4 or IPv6.
+- `vip` (Boolean)
```

Comment:
The `version` and `vip` fields no longer mention the default value. Possibly we just need to add it to the description for each field in the schema?